stdlib/runtime-api/src/main/ballerina/src/runtime/Module.md

```diff
@@ -1,122 +1,65 @@
-        authContext.authToken = "abc.pqr.xyz";
-    }
+    // Set the claims.
+    map<any> claims = { email: "tom@ballerina.com", org: "wso2" };
+    principal.claims = claims;
 
-    // Retrieve data from the invocation context.
-    runtime:InvocationContext invocationContext1 = runtime:getInvocationContext();
-    runtime:Principal? principal1 = invocationContext1["principal"];
-    if (principal1 is runtime:Principal) {
-        // Retrieve user name.
-        string? userName = principal1["username"];
-        io:println(userName);
-
-        // Retrieve claims.
-        map<any>? retrievedClaims = principal1["claims"];
-        io:println(retrievedClaims);
-
-        // Retrieve scopes.
-        string[]? retrievedScopes = principal1["scopes"];
-        io:println(retrievedScopes);
-    }
-
-    runtime:AuthenticationContext? authContext1 = 
-                            invocationContext1["authenticationContext"];
-    if (authContext1 is runtime:AuthenticationContext) {
-        // Retrieve auth scheme.
-        string? authScheme = authContext1["scheme"];
-        io:println(authScheme);
-
-        // Retrieve auth token.
-        string? token = authContext1["authToken"];
-        io:println(token);
-    }
+    // Set scopes.
+    string[] scopes = ["email", "profile"];
+    principal.scopes = scopes;
 }
-```
 
-The following sample shows how to halt the current `worker` for a given time period.
-```ballerina
-import ballerina/runtime;
+runtime:AuthenticationContext? authContext = invocationContext["authenticationContext"];
+if (authContext is runtime:AuthenticationContext) {
+    // Set the auth scheme.
+    authContext.scheme = "jwt";
 
-// Sleep the current worker for 5 seconds.
-runtime:sleep(5000);
+    // Set the auth token.
+    authContext.authToken = "abc.pqr.xyz";
+}
 ```
 
-The following sample shows how to access properties from the runtime. 
+The following code snippet shows how to access the `runtime:InvocationContext` and get the data.
 ```ballerina
-import ballerina/runtime;
-
-// Retrieve the property ‘ballerina version’ from the runtime.
-runtime:getProperty("ballerina.version");
-```
-
-### CallStack Element
-
-The runtime module includes a utility method to retrieve the current call stack and the particular call stack frame for
- an error. 
-
-The following sample shows how to access the call stack and how to trap an error.
-
-```ballerina
-import ballerina/io;
-import ballerina/runtime;
-
-public function main() {
-    // Print the current call stack.
-    io:println(runtime:getCallStack());
-    var errorMessage = trap getError();
-    if (errorMessage is error) {
-        io:println(errorMessage.reason());
-    }
+runtime:InvocationContext invocationContext = runtime:getInvocationContext();
+runtime:Principal? principal = invocationContext["principal"];
+if (principal is runtime:Principal) {
+    // Retrieve the user name.
+    string? userName = principal["username"];
+    io:println(userName);
```

Comment:
Shall we remove `io:println` from the docs. add only one related to the module

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/22534/commits/ba874a5eec5ea9827e2778a1fc28087e9160ffc6